cmd/api/src/api/middleware/auth.go

```diff
@@ -67,7 +67,7 @@ func AuthMiddleware(authenticator api.Authenticator) mux.MiddlewareFunc {
 				switch authScheme {
 				case api.AuthorizationSchemeBearer:
 					if userAuth, err := authenticator.ValidateSession(request.Context(), schemeParameter); err != nil {
-						api.WriteErrorResponse(request.Context(), api.BuildErrorResponse(http.StatusUnauthorized, api.ErrorResponseDetailsAuthenticationInvalid, request), response)
+						api.WriteErrorResponse(request.Context(), api.BuildErrorResponse(http.StatusUnauthorized, err.Error(), request), response)
```

Comment:
We should revert this change. Any time we return a 401 Unauthorized we have to be careful so not to disclose any information that could be potentially leveraged by a bad actor.